modules/static/googlecloudplatform/bq-schema-api/initref

```diff
@@ -1 +1 @@
-31a7e43419f7c19d79de0bb506798bc602287e80
+v1.1.0
```

Comment:
This is the closest tag:

```
➜  protoc-gen-bq-schema git:(31a7e43) git describe --all 31a7e43419f7c19d79de0bb506798bc602287e80
tags/v1.1.0
```

Here are all the releases: https://github.com/GoogleCloudPlatform/protoc-gen-bq-schema/releases

Reply:
cc @mfridman since you are the one who added this module, just FYI.

Reply:
That seems about right, I primarily asked the upstream project to pin a tag so we could use the plugin in that repository, less about the module and more about the plugin.

https://github.com/GoogleCloudPlatform/protoc-gen-bq-schema/pull/46

Not sure if there's a question here.

Reply:
protoc-gen-bq-schema will move to semver in another PR.